src/styles/Base/variables.scss

```diff
@@ -1,10 +1,13 @@
+/* Fonts */
 $font-text: 'Roboto Condensed', sans-serif;
 $font-heading: 'PT Serif', serif;
 $font-links: 'Montserrat', sans-serif;
 
 /* Colors */
-
 $black: #1F1C17;
 $yellow: #FFA10C;
 $brown: #593C1A;
 $white: #FAFAFA;
+
```

Comment:
Just FYI, this will result in to a conflict, as I have added more colours this morning, but have not yet merged my branch.